environment.yml

```
@@ -4,24 +4,25 @@ channels:
  - anaconda
  - defaults
 dependencies:
+- gdal
+- rasterio
+- affine
+- Fiona
```

```suggestion
- fiona
```
Conda package name is lowercase, but it looks like it works either way